pom.xml

```diff
@@ -1816,6 +1775,104 @@ This will force full Scala code rebuild in downstream modules.
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>get-buildvers</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <script language="jython">
+import ConfigParser
+config = ConfigParser.ConfigParser()
+config.read("release.properties")
+<!-- #if scala-2.12 -->
+print(config.get("Scala212ReleaseSection", "${expression}"))
+<!-- #endif scala-2.12 -->
+<!-- #if scala-2.13 --><!--
+print(config.get("Scala213ReleaseSection", "${expression}"))
+--><!-- #endif scala-2.13 -->
+                                </script>
+                            </target>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>create-release-properties</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>initialize</phase>
+                        <configuration>
+                            <target>
+                                <script language="jython">
+import subprocess
+import os
+<!-- #if scala-2.12 -->
+pom = ""
+<!-- #endif scala-2.12 -->
+<!-- #if scala-2.13 --><!--
+pom = "-f scala2.13"
+--><!-- #endif scala-2.13 -->
+
+# Define the Bash command
+bash_command = """
+export TEMP=$(mvn help:all-profiles -pl . {pom}| sort | uniq | awk "/release[0-9]/ {{print substr(\$3, 8)}}");
```

Comment:
Can we have a standard execution inside pom https://maven.apache.org/plugins/maven-help-plugin/all-profiles-mojo.html instead of calling ant / jython to call a bash script with Maven